x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/tour_step.tsx

```diff
@@ -11,26 +11,53 @@ import type { EuiTourStepProps } from '@elastic/eui';
 import { EuiButton, EuiImage, EuiSpacer, EuiText, EuiTourStep } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 
+import styled from 'styled-components';
+import { useShallowEqualSelector } from '../../hooks/use_selector';
+import { TimelineId } from '../../../../common/types';
+import { timelineDefaults } from '../../../timelines/store/timeline/defaults';
+import { timelineSelectors } from '../../../timelines/store/timeline';
 import { useTourContext } from './tour';
-import { securityTourConfig, SecurityStepId } from './tour_config';
+import { SecurityStepId, securityTourConfig } from './tour_config';
+
 interface SecurityTourStep {
   children?: React.ReactElement;
   step: number;
   stepId: SecurityStepId;
 }
 
+const isStepExternallyMounted = (stepId: SecurityStepId, step: number) =>
+  step === 5 && stepId === SecurityStepId.alertsCases;
+
+const StyledTourStep = styled(EuiTourStep)<EuiTourStepProps & { stepId: SecurityStepId }>`
+  &.euiPopover__panel[data-popover-open] {
+    z-index: ${({ step, stepId }) =>
+      isStepExternallyMounted(stepId, step) ? '9000 !important' : '1000 !important'};
```

Comment:
I needed to lower the `z-index` to prevent bugs like this: 
<img width="1294" alt="Screen Shot 2022-10-28 at 10 14 47 AM" src="https://user-images.githubusercontent.com/6935300/198684402-b0ddbcd1-e4ae-404b-9214-e373be985b68.png">

But it needs to be high again for the cases step or else:
<img width="1290" alt="Screen Shot 2022-10-28 at 10 15 14 AM" src="https://user-images.githubusercontent.com/6935300/198684485-fcb7e6fb-b532-425e-b884-ec795a7019be.png">